usr/share/rear/conf/default.conf

```
@@ -276,13 +276,16 @@ test "$USER_INPUT_TIMEOUT" || USER_INPUT_TIMEOUT=300
 # when a predefined input value is specified for a particular UserInput() call
 # via a matching USER_INPUT_user_input_ID variable (see below).
 # The minimum waiting time to interrupt an automated input is one second.
-# The default is 10 seconds to give the user a better chance to recognize
-# an automated input and be able to actually hit a key to interrupt.
+# The default interrupt timeout must be sufficiently long for the user
+# to read and understand the possibly unexpected UserInput() message
+# and then some more time to make a decision whether or not
+# the automated action is actually the right one and
+# if not a bit more time to hit a key to interrupt.
 # USER_INPUT_INTERRUPT_TIMEOUT is set to a default value here only
 # if not already set so that the user can set it also like
-#   export USER_INPUT_INTERRUPT_TIMEOUT=5
+#   export USER_INPUT_INTERRUPT_TIMEOUT=10
 # directly before he calls "rear ...":
-test "$USER_INPUT_INTERRUPT_TIMEOUT" || USER_INPUT_INTERRUPT_TIMEOUT=10
+test "$USER_INPUT_INTERRUPT_TIMEOUT" || USER_INPUT_INTERRUPT_TIMEOUT=30
```

Can we start to use this Bash syntax consistently?

```suggestion
USER_INPUT_INTERRUPT_TIMEOUT="${USER_INPUT_INTERRUPT_TIMEOUT:-30}"
```

---

Done via
https://github.com/rear/rear/pull/2981/commits/a436ad55e5dc437f173eb3c253341fd4d3c59551